teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -131,194 +134,110 @@ surfacing the verification information and building a corpus of verified resourc
 
 ## Definitions
 
-The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”,
-“SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and
-“OPTIONAL” are to be interpreted as described in [RFC 2119][rfc2119].
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, 
+“NOT RECOMMENDED”, “MAY”, and “OPTIONAL” are to be interpreted as described in [RFC 2119][rfc2119].
 
 Terms used in this TEP are defined as follows:
-* **Catalog**: A repository that complies with the organization contract
-  defined in [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+
+* **Catalog**: A repository that complies with the organization contracts defined in [TEP-0003][tep-0003-org] or
+[TEP-0115][tep-0115]. The git-based versioning defined in [TEP-0115][tep-0115] is preferred. 
+
 * **Resource**: Item shared in a Tekton Catalog e.g. `Task` or `Pipeline`.
-* **Tekton Catalog Maintainers**: The core group of OWNERS who can approve
-  changes in Tekton Catalogs. They are defined in [OWNERS][catalog-owners]
-  file in the existing Catalog at `tektoncd/catalog`.
+
+* **Tekton Catalog Maintainers**: The core group of OWNERS who can approve changes in verified Tekton Catalogs.
+Today, they are defined in [OWNERS][catalog-owners] file in the existing Catalog at `tektoncd/catalog`.
```

Comment:
What about maintainers of the `tektoncd-catalog` GitHub org ?

Reply:
Thanks, added maintainers update